crates/autopilot/src/infra/mod.rs

```diff
@@ -1,2 +1,5 @@
 pub mod blockchain;
+pub mod orderbook;
 pub mod persistence;
+
+pub use orderbook::Orderbook;
```

Comment:
I'd make this `infra::shadow::Orderbook`

Reply:
fixed, referred with `infra::shadow::Orderbook` in the Runloop